packages/handlersjs-http/lib/handlers/http-cors-request.handler.ts

```diff
@@ -33,6 +34,7 @@ export class HttpCorsRequestHandler extends HttpHandler {
 
   constructor(
     private handler: HttpHandler,
+    private errorHandler: ErrorHandler,
```

Comment:
On second thought, I'd not pass this as a constructor argument. This class already takes a nested handler, so we can simple construct a handler with an error handler in the configuration (i.e. a pipe-through handler of the real/previous nested handler followed by an error handler).

Reply:
True. I did not think of a pipe-through handler.